### PR TITLE
EZP-30395: Move and rename templates

### PR DESCRIPTION
**Pull request by DominikaK**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | https://jira.ez.no/browse/EZP-30395
| Versions      | master (3.0)

### CODE REVIEW

File: docs/cookbook/paginating_api_search_results.md [hunk 1]
```diff
@@ -60,13 +60,13 @@ class ArticleListController extends Controller
         <ul>
         {# Loop over the page results #}
         {% for article in pagerArticle %}
-            <li><a href={{ path('ez_urlalias', {'contentId': article.contentInfo.id}) }}>{{ez_content_name( article ) }}</a></li>
+            <li><a href={{ path('ez_urlalias', {'contentId': article.content_info.id}) }}>{{ez_content_name( article ) }}</a></li>
```
Comment (pawbuj) [inline docs/cookbook/paginating_api_search_results.md]: the previous version is correct.

File: docs/cookbook/paginating_api_search_results.md [hunk 2]
```diff
@@ -60,13 +60,13 @@ class ArticleListController extends Controller
...
         {% endfor %}
         </ul>
     </div>
  
     {# Only display Pagerfanta navigator if needed. #}
-    {% if pagerArticle.haveToPaginate() %}
+    {% if pagerArticle.have_to_paginate() %}
```
Comment (pawbuj) [inline docs/cookbook/paginating_api_search_results.md]: the previous version is correct.

File: docs/guide/templates.md [hunk 3]
```diff
@@ -242,7 +242,7 @@ The *translated name* is held in a `VersionInfo` object, in the `names` prope
 
 {# In earlier versions the same is available using Twig translation helper ez_content_name() #}
 <h2>Translated Content name: {{ ez_content_name( content ) }}</h2>
-<h3>Also works from ContentInfo: {{ ez_content_name( content.contentInfo ) }}</h3>
+<h3>Also works from ContentInfo: {{ ez_content_name( content.content_info ) }}</h3>
```
Comment (pawbuj) [inline docs/guide/templates.md]: the previous version is correct.

File: docs/guide/templates.md [hunk 4]
```diff
@@ -259,7 +259,7 @@ You can also **force a locale** in a second argument:
     This property is the actual Content name, but **in the main language only** (so it is not translated).
 
     ``` html+twig
-    <h2>Content name: {{ content.contentInfo.name }}</h2>
+    <h2>Content name: {{ content.content_info.name }}</h2>
```
Comment (pawbuj) [inline docs/guide/templates.md]: the previous version is correct.

File: docs/guide/twig_functions_reference.md [hunk 5]
```diff
@@ -39,7 +39,7 @@ If languages were specified during retrieval of Content object, you can render n
 
 `ez_content_name ( eZ\Publish\API\Repository\Values\Content\Content content [, string forcedLanguage ] ) : string`
 
-`ez_content_name ( contentInfo [, string forcedLanguage ] ) : string`
+`ez_content_name ( content_info [, string forcedLanguage ] ) : string`
```
Comment (pawbuj) [inline docs/guide/twig_functions_reference.md]: the previous version is correct.

File: docs/guide/twig_functions_reference.md [hunk 6]
```diff
@@ -50,7 +50,7 @@ If languages were specified during retrieval of Content object, you can render n
 
 ``` html+twig
 <h2>Content name in current language: {{ ez_content_name( content ) }}</h2>
-<h2>Content name in current language, from ContentInfo: {{ ez_content_name( content.contentInfo ) }}</h2>
+<h2>Content name in current language, from ContentInfo: {{ ez_content_name( content.content_info ) }}</h2>
```
Comment (pawbuj) [inline docs/guide/twig_functions_reference.md]: the previous version is correct.

File: docs/guide/twig_functions_reference.md [hunk 7]
```diff
@@ -248,7 +248,7 @@ The formats are defined in [user preferences](config_back_office.md#date-and-tim
 The filters accept `\DateTimeInterface` as argument.
 If the argument is null, the filter returns the current date and time in the selected format.
 
-For example `{{ contentInfo.publishedDate|ez_full_datetime }}` will return `03 May 2019 23:03`.
+For example `{{ content_info.publishedDate|ez_full_datetime }}` will return `03 May 2019 23:03`.
```
Comment (pawbuj) [inline docs/guide/twig_functions_reference.md]: here we do not know what will be the name of the variable. Its just an example.

File: docs/guide/twig_functions_reference.md [hunk 8]
```diff
@@ -486,7 +486,7 @@ The formats are defined in [user preferences](config_back_office.md#date-and-tim
...
```
Comment (pawbuj) [inline docs/guide/twig_functions_reference.md]: same here, it is just an example , both version are correct

File: docs/tutorials/extending_admin_ui/3_creating_a_content_list.md [hunk 9]
```diff
@@ -183,10 +183,10 @@ Finally, create an `all_content_list.html.twig` file in `src/EzSystems/Extending
             <tbody>
             {% for article in articles %}
                 <tr>
-                    <td><a href={{path('ez_urlalias', {'contentId': article.contentInfo.id})}}>{{ ez_content_name(article.contentInfo) }}</a></td>
-                    <td>{{ article.contentInfo.contentTypeId }}</td>
-                    <td>{{ article.contentInfo.modificationDate|ez_full_datetime }}</td>
-                    <td>{{ article.contentInfo.publishedDate|ez_full_datetime }}</td>
+                    <td><a href={{path('ez_urlalias', {'contentId': article.content_info.id})}}>{{ ez_content_name(article.content_info) }}</a></td>
+                    <td>{{ article.content_info.contentTypeId }}</td>
```
Comment (pawbuj) [inline docs/tutorials/extending_admin_ui/3_creating_a_content_list.md]: previous version of this block of code is correct

File: docs/tutorials/platform_beginner/4_display_a_list_of_content_items.md [hunk 10]
```diff
@@ -159,7 +159,7 @@ Because this template will be rendered inside a table, it starts with a `<tr>` t
 ``` html+twig
 <tr>
     <td>
-        <a href="{{ path( "ez_urlalias", { 'locationId': content.contentInfo.mainLocationId } ) }}"
+        <a href="{{ path( "ez_urlalias", { 'locationId': content.content_info.mainLocationId } ) }}"
```
Comment (pawbuj) [inline docs/tutorials/platform_beginner/4_display_a_list_of_content_items.md]: prev is correct

File: docs/tutorials/platform_beginner/4_display_a_list_of_content_items.md [hunk 11]
```diff
@@ -117,7 +117,7 @@ The `<head>` of the `<table>` is contained in this **Ride** list template, while
             <p>{{ 'There are no Rides yet.'|trans }}</p>
         {% endif %}
 
-        {% if pagerRides.haveToPaginate() %}
+        {% if pagerRides.have_to_paginate() %}
```
Comment (pawbuj) [inline docs/tutorials/platform_beginner/4_display_a_list_of_content_items.md]: prev is correct

File: docs/tutorials/extending_admin_ui/4_filtering_the_content_list.md [hunk 12]
```diff
@@ -186,10 +186,10 @@ inside `<section class="container my-4">`:
                 <tbody>
                 {% for article in articles %}
                     <tr>
-                        <td><a href={{path('ez_urlalias', {'contentId': article.contentInfo.id})}}>{{ ez_content_name(article.contentInfo) }}</a></td>
-                        <td>{{ article.contentInfo.contentTypeId }}</td>
-                        <td>{{ article.contentInfo.modificationDate|ez_full_datetime }}</td>
-                        <td>{{ article.contentInfo.publishedDate|ez_full_datetime }}</td>
+                        <td><a href={{path('ez_urlalias', {'contentId': article.content_info.id})}}>{{ ez_content_name(article.content_info) }}</a></td>
+                        <td>{{ article.content_info.contentTypeId }}</td>
```
Comment (pawbuj) [inline docs/tutorials/extending_admin_ui/4_filtering_the_content_list.md]: previous version is corrent, all 5 lines

**Comment by DominikaK** [comment]
> you can also mention somewhere that `pagelayout` is deprecated but will be also work like `page_layout`

Deprecation info will be done in a separate PR, with a whole list of BC breaks/deprecations for 3.0

**Pull request merged**